lighthouse-plugin-ad-speed-insights/audits/ad-blocking-tasks.js

```diff
@@ -15,6 +15,7 @@
 const MainThreadTasks = require('lighthouse/lighthouse-core/computed/main-thread-tasks');
 const NetworkRecords = require('lighthouse/lighthouse-core/computed/network-records');
 const {auditNotApplicable} = require('../utils/builder');
+const {AUDITS, NOT_APPLICABLE} = require('../messages/en-US.js');
```

Comment:
Is it possible to avoid hardcoding `en-US` since it will be changed eventually?

Even if the middle layer is trivial, can we get started with some of the scaffolding?

Reply:
Feel free to address this in a follow-up if you have some thoughts. Perhaps an I2I is warranted here